### PR TITLE
5111 fix sankey pointer

### PR DESCRIPTION
**Pull request by brett-varney**
https://federal-spending-transparency.atlassian.net/browse/DA-5111

Show cursor pointer on nodes. Also the path tool tips (titles) are showing, just not hovering over the actual node text. This is different from prod, but I have no idea why.

### CODE REVIEW

**Comment by anjenkin** [comment]
@brett-varney Pointer cursor shows when hovering over the object nodes.  Pointer cursor doesn't show when hovering over ticket as expected.

**Comment by brett-varney** [comment]
> 
> 
> @brett-varney Pointer cursor shows when hovering over the object nodes. Pointer cursor doesn't show when hovering over ticket as expected.

True, that was changed in the comments. Sorry the AC wasn't updated.

**Comment by anjenkin** [comment]
@brett-varney Hey Brett, when I checked earlier today the text part was in the AC.  "On desktop, the mouse cursor changes to a hand pointer (instead of an arrow) when hovering over the Budget Function, Object Class nodes and **Text in the Sankey visualization**."  My JIRA is weird now so I can check to see when this was updated.

**Comment by brett-varney** [comment]
> 
> 
> @brett-varney Hey Brett, when I checked earlier today the text part was in the AC. "On desktop, the mouse cursor changes to a hand pointer (instead of an arrow) when hovering over the Budget Function, Object Class nodes and **Text in the Sankey visualization**." My JIRA is weird now so I can check to see when this was updated.

You're right, the text part was in the AC. The comments changed that, but the ticket body wasn't updated to match. But it appears there's new discussion anyway...

**Comment by brett-varney** [comment]
@anjenkin Pointer also shows on text only (not entire block as before), so tips show. I think this is what they want, but getting verification.

**Comment by brett-varney** [comment]
@anjenkin I clarified the AC based on ticket comments and slack public and private messages.

**Comment by anjenkin** [comment]
@brett-varney Hmm.. now the tooltip with the mapping and obligation isn't showing when hovering over the text.  Per Beth's comment it sounds like she would really like the tooltip to show.  If for some reason we can't get both the tooltip and the hand cursor to show, Beth should be notified.

**Comment by brett-varney** [comment]
> 
> 
> @brett-varney Hmm.. now the tooltip with the mapping and obligation isn't showing when hovering over the text. Per Beth's comment it sounds like she would really like the tooltip to show. If for some reason we can't get both the tooltip and the hand cursor to show, Beth should be notified.

I don't read it that way, and it can't. Each node and label pertain to several paths, so there's no way to determine which one they want, unless it's the one directly underneath the label, which will be very difficult to determine if the entire label is all within one pathway, which many aren't.

**Comment by anjenkin** [comment]
I'm not sure if we saying the same thing.  We have the code to add the correct tooltips on hover of the label.  This should be it -     link.append("title").text((d) => `${d.source.name} → ${d.target.name}\n${format(d.value)}`);

2ac1ea8f-c547-43d9-8375-b648c88d31d3

**Comment by brett-varney** [comment]
> 
> 
> I'm not sure if we saying the same thing. We have the code to add the correct tooltips on hover of the label. This should be it - link.append("title").text((d) => `${d.source.name} → ${d.target.name}\n${format(d.value)}`); It works in prod. We just need both the tooltip and the hand cursor. Let me know if we need to set up a meeting to discuss further.
> 
> 2ac1ea8f-c547-43d9-8375-b648c88d31d3

So, hovering over General Government should show which destination? The top one? The largest one? Alphabetical?

**Comment by anjenkin** [comment]
In prod it's showing the target and obligation for the largest one

**Comment by anjenkin** [comment]
If you're unsure, please clarify with Beth and / or Kelli

**Comment by brett-varney** [comment]
> 
> 
> If you're unsure, please clarify with Beth and / or Kelli

I did, I'm not now.

**Comment by brett-varney** [comment]
> 
> 
> In prod it's showing the target and obligation for the largest one

That's not consistently true

image

**Comment by brett-varney** [comment]
@anjenkin Since this may not get merged today, would you please let Ritesh know when he can run his tests?

File: src/components/visualizations/sankey/sankey.jsx [hunk 1]
```diff
@@ -520,10 +518,10 @@ function Sankey(props) {
         return `link-${i}`;
       })
       .style("stroke-width", (d) => Math.max(1, d.dy))
-      .sort((a, b) => b.dy - a.dy);
+      .sort((a, b) => b.dy - a.dy)
+      ;
 
-    link.append("title")
-      .text((d) => `${d.source.name} → ${d.target.name}\n${format(d.value)}`);
+    link.append("title").text((d) => `${d.source.name} → ${d.target.name}\n${format(d.value)}`);
```
Comment (anjenkin) [inline src/components/visualizations/sankey/sankey.jsx]: I guess this was in prod?   It's not working I guess as the tooltips aren't showing.  Now that we've determined we don't want to show the tooltip, I guess it's fine.

**Comment by anjenkin** [comment]
will merging after testing the build

**Pull request merged**